lua/weapons/weapon_laserpointer.lua

```diff
@@ -0,0 +1,326 @@
+-- This file is subject to copyright - contact swampservers@gmail.com for more information.
+
+SWEP.PrintName = "Laser Pointer"
+SWEP.Author = "Britain1944"
+SWEP.Instructions = "Left click: Turn on and off the laser.\nRight click: Change laser power\nRepurchase for more ammo."

```

Comment:
Can it be hold down to use the laser? you can use SWEP.Automatic = true and some kind of timer logic to disable it when the weapon hasn't been fired in the past 0.1 second

Reply:
Yes the weapon is an automatic weapon, so it turns on when you shoot and turns off when you're not shooting

Reply:
but I don't think the weapon needs the second part

Reply:
the instructions imply it's a toggle to turn it on and off, i think it should be you hold it down to keep it on

Reply:
Yeah lemme change the instructions, it doesn't say toggle but it doesn't say that you hold it down for laser